modules/Bio/EnsEMBL/Compara/PipeConfig/Parts/DumpConservationScores.pm

```diff
@@ -17,9 +17,10 @@ limitations under the License.
 
 =cut
 
-=head1 SYNOPSIS
+=head1 DESCRIPTION
 
-Pipeline to dump conservation scores as bedGraph and bigWig files
+    This is a partial PipeConfig for most part of the pipeline that dumps the
+    conservation scores as bedGraph and bigWig files.
```

Comment:
I'm not sure about this description - can we change it?
> This PipeConfig contains the core analyses required to dump the conservation scores as bedGraph and bigWig files.

Reply:
constrained elements as bigBed and conservation scores as bigWig